frontend/src/locales/en/club.json

```
@@ -17,12 +18,20 @@
+      "terms_prefix": "I declare that I have read and accept the terms presented in the",
+      "terms_html": "I declare that I have read and accept the terms presented in the <a href='/club-rules' class='underline'>SGG Club Regulations</a>"
     },
+    "submit": "Join SGG Club",
     "titles": {
       "main": "Fill in the form to refer a member:"
     }
   },
+  "story": {
+    "not_found": "Story not found or no longer available."
+  },
+  "stories": {
```

Biome (2.1.2)

[error] The key stories was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)